postgresql-async/build.gradle.kts

```diff
@@ -24,10 +25,12 @@ dependencies {
     compile("io.netty:netty-transport:$NETTY_VERSION")
     compile("io.netty:netty-handler:$NETTY_VERSION")
     compile("io.github.microutils:kotlin-logging:$KOTLIN_LOGGING_VERSION")
+    compile("com.ongres.scram:client:$SCRAM_CLIENT_VERSION")
     testImplementation("junit:junit:$JUNIT_VERSION")
     testImplementation("org.jetbrains.kotlin:kotlin-test-junit:$KOTLIN_VERSION")
     testImplementation("org.jetbrains.kotlin:kotlin-test-junit:$KOTLIN_VERSION")
     testImplementation("org.assertj:assertj-core:$ASSERTJ_VERSION")
+    testImplementation("org.jetbrains.kotlin:kotlin-reflect:$KOTLIN_VERSION")
```

Comment:
Solve a compiler warning about kotlin-reflect being 1.3.x while the rest of kotlin support is 1.4.x. Note, this warning appears for other projects (such as mysql's client) as well but I didn't want to be presumptuous in proliferating this fix.